### PR TITLE
chore(addons): copilot command rename

### PR DESCRIPTION
**Pull request by iamhopaul123**
Renaming for `addons` package. Will be including a manual test soon.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: templates/addons/cf.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
 AWSTemplateFormatVersion: 2010-09-09
-Description: Additional resources for application '{{.AppName}}'
+Description: Additional resources for application '{{.SvcName}}'
```
Comment (bvtujo) [inline templates/addons/cf.yml:4]:
```suggestion
Description: Additional resources for service '{{.SvcName}}'
```

Comment (iamhopaul123) [reply templates/addons/cf.yml:4]: oh good catch

**Comment by iamhopaul123** [comment]
Nested stack deployed, indicating manual test pass.

File: internal/pkg/addons/testdata/template.yml [hunk 2]
```diff
@@ -1,19 +1,19 @@
 Parameters:
-  Project:
+  App:
     Type: String
```
Comment (efekarakus) [inline internal/pkg/addons/testdata/template.yml:3]: Is this test file used anywhere?

Comment (efekarakus) [reply internal/pkg/addons/testdata/template.yml:3]: Also I wonder if we should first do the translation of "app -> svc" and then tackle "project -> app" to reduce the scope of the changes.

Comment (iamhopaul123) [reply internal/pkg/addons/testdata/template.yml:3]: in output_test I think

Comment (efekarakus) [reply internal/pkg/addons/testdata/template.yml:3]: oh i was confused because i didn't see any changes to `Project` in the PR, yet the tests pass 🤔

Comment (efekarakus) [reply internal/pkg/addons/testdata/template.yml:3]: ignore thi

> Also I wonder if we should first do the translation of "app -> svc" and then tackle "project -> app" to reduce the scope of the changes.

ignore this comment, i think what's here is nice. It's okay if the CLI itself doesn't work while we are on this branch but the unit tests must pass for each PR 👍

**Pull request merged**